autodistill/text_classification/text_classification_base_model.py

```diff
@@ -0,0 +1,24 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+
+from autodistill.core import BaseModel
+
+from .text_classification_ontology import TextClassificationOntology
+
+@dataclass
+class TextClassificationBaseModel(BaseModel):
+    ontology: TextClassificationOntology
+
+    @abstractmethod
+    def predict(self, input: str) -> dict:
+        pass
+
+    def label(
+        self,
+        input_jsonl: str,
```

Comment:
Is this supposed to be a file path or the string contents?

Reply:
This will be a file path. We could add support for loading in jsonl from strings, but that can be in a future release if there is demand.